src/shared_modules/content_manager/doc/components/ZIP_DECOMPRESSOR.md

```diff
@@ -0,0 +1,38 @@
+# Zip Decompressor stage
+
+## Details
+
+The [zip decompressor](../../src/components/zipDecompressor.hpp) stage is part of the Content Manager orchestration and is in charge of decompressing the `.zip` files fetched in the download stage. If the decompression is successful, this stage also updates the context [data paths](../../src/components/updaterContext.hpp).
+
+### Paths update example
+
+Paths before stage execution:
+```json
+"paths": [
+    "/tmp/outputFolder/downloads/file.zip"
+]
+```
+
+`file.zip` content:
+```bash
+|-- data
+|   |-- file_b.xml
+|   `-- file_c.xml
+`-- file_a.json
+```
+
+Paths after stage execution:
+```json
+"paths": [
+    "/tmp/outputFolder/file_a.json",
+    "/tmp/outputFolder/data/file_b.xml",
+    "/tmp/outputFolder/data/file_c.xml"
+]
+```
```

Comment:
The files are at `contents/` right?
```suggestion
Paths after stage execution:
```json
"paths": [
    "/tmp/outputFolder/contents/file_a.json",
    "/tmp/outputFolder/contents/data/file_b.xml",
    "/tmp/outputFolder/contents/data/file_c.xml"
]
```
```

Reply:
Yes! Fixed.